exercises/03-sagas/after/Divergent.Shipping/Divergent.Shipping.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus" Version="7.0.1" />
+    <PackageReference Include="NServiceBus.Persistence.Sql.MsBuild" Version="4.2.0" />
```

Comment:
Is this reference required in this project (and in other similar places)?

Reply:
It seems we need to have them to get the build to work